CHANGELOG.md

```diff
@@ -4,11 +4,12 @@
 
 * Fixed HMR in `piral-cli-webpack5` when running `piral debug`
 * Updated to use React 17 (#312)
+* Adding `feed` parameter to `piral debug`
```

Comment:
`pilet debug`, right?

Reply:
yes 👍 
even had that change already done locally but forgot to push it - will do it now